websocket/integrationTests/clientAndServer_test.go

```diff
@@ -107,7 +108,7 @@ func TestStartServerStartClientCloseServer(t *testing.T) {
 
 	numMessagesReceived := 0
 	payloadHandler := &testscommon.PayloadHandlerStub{
-		ProcessPayloadCalled: func(payload []byte, _ string) error {
+		ProcessPayloadCalled: func(payload []byte, _ string, _ uint32) error {
```

Comment:
Perhaps keep consistency in this file with named/unnamed parameter

Reply:
changed to use named parameter